src/corner_kick/src/components/LayersPanel/__stories__/index.tsx

```diff
@@ -18,23 +18,23 @@ stories.add('with layers', () => (
         <LayersPanel
             layers={[
                 {
-                    id: 'friendly_robots',
                     name: 'Friendly Robots',
```

Comment:
Are these the layer names from messages? 

If so, I currently have the following layer names defined on VM: "field", "ball", "ball_vel", "fiendly_robot", "fiendly_robot_vel", "enemy_robot", "enemy_robot_vel", "nav", "rule", "passing", "test", "misc". (util/visualizer_messenger/visualizer_messenger.cpp)

We either needs to have the same naming, or have some 1:1 mapping between layer name from the message, and layer name for UI.